Automattic/jetpack
# Licensing: Move image asset to package

#### Changes proposed in this Pull Request:

* Moves images that are used for licensing components to licensing package to minimize external dependencies

#### Jetpack product discussion



No existing conversation, though it's to move the licensing activation UI from the Jetpack plugin to My Jetpack. This small refactor will make it easier to use the components in My Jetpack by minimizing the amount of data has to be sent to the ActivationScreen component.

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






* Checkout PR
* `jetpack build plugins/jetpack`
* Go to `$site.com/wp-admin/admin.php?page=jetpack#/license/activation`
* Ensure that UI loads and no errors
* Inspect image source and verify that `_inc/build` included in that source
* Enter license key
* Ensure that UI changes and success shown
* Verify no errors
* Inspect image source and verify that `_inc/build` included in that source
